### PR TITLE
Feat: add a validation to the password field

### PR DESCRIPTION
**Pull request by GusBaamonde**
**Add a validation to the password field:**

1. 8 characters minimum
2. One letter
3. One number

**it adjusts to the mobile view.**

message-password

### CODE REVIEW

File: assets/templates/signup.html [hunk 1]
```diff
@@ -110,9 +110,9 @@ <h5>email, automation & data marketing</h5>
...
                 <div class="wrapper-password">
-                  <p class="password-message"><span class="waiting-message">8 caracteres como mínimo</span><span class="waiting-message">Un dígito</span></p>
-                  <!-- <p class="password-message"><span class="lack-message">8 caracteres como mínimo</span><span class="lack-message">Un dígito</span></p>
-                  <p class="password-message"><span class="complete-message">8 caracteres como mínimo</span><span class="complete-message">Un dígito</span></p>
+                  <p class="password-message"><span class="waiting-message">8 caracteres como mínimo</span><span class="waiting-message">Una letra</span><span class="waiting-message">Un número</span></p>
+                  <!-- <p class="password-message"><span class="lack-message">8 caracteres como mínimo</span><span class="lack-message">Una letra</span><span class="lack-message">Un número</span></p>
+                  <p class="password-message"><span class="complete-message">8 caracteres como mínimo</span><span class="complete-message">Una letra</span><span class="complete-message">Un número</span></p>
```
Comment (andresmoschini) [inline assets/templates/signup.html]: Does it require changes in webapp?

Comment (GusBaamonde) [reply assets/templates/signup.html]: Ok, I do not know where those messages are, but I can add them if you tell me where

Comment (andresmoschini) [reply assets/templates/signup.html]: @cbernat has a related PR open, if it requires changes in webapp markup, please let us know.

Comment (GusBaamonde) [reply assets/templates/signup.html]: @cbernat implements it!

Comment (cbernat) [reply assets/templates/signup.html]: Done in https://github.com/FromDoppler/doppler-webapp/pull/126

**Pull request merged**